sdk/src/jumbf_io.rs

```diff
@@ -67,28 +55,6 @@ pub fn get_assetio_handler(ext: &str) -> Option<Box<dyn AssetIO>> {
     }
 }
 
-#[cfg(target_arch = "wasm32")]
-pub fn get_assetio_handler(ext: &str) -> Option<Box<dyn AssetIO>> {
```

Comment:
Where did this function go?

Reply:
These were duplicates only created so that we could include pdf support, but we removed that, so they were really just entirely duplicate functions, so there was no longer a need to have wasm versions.